edit-post/store/selectors.js

```diff
@@ -108,6 +108,23 @@ export function isFeatureActive( state, feature ) {
 	return !! state.preferences.features[ feature ];
 }
 
+/**
+ * Returns true if the the plugin item is pinned to the header.
+ * When the value is not set it defaults to true.
+ *
+ * @param  {Object}  state      Global application state.
+ * @param  {string}  pluginName Plugin item name.
+ * @return {boolean}            Whether the plugin item is pinned.
+ */
+export function isPluginItemPinned( state, pluginName ) {
+	const defaultValue = true;
+	const pinnedPluginItems = getPreference( state, 'pinnedPluginItems', {} );
+
+	return isUndefined( pinnedPluginItems[ pluginName ] ) ?
+		defaultValue :
+		Boolean( pinnedPluginItems[ pluginName ] );
```

Comment:
When are we expecting the value to be anything other than a boolean here to warrant the coercion?

Reply:
I don’t think so. 

I also agree that get from lodash will be easier to read in both places. I forgot about this fact that it check if value is defined 😎